src/OpenTelemetry.Abstractions/OpenTelemetry.Abstractions.csproj

```diff
@@ -1,33 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenTelemetry.sln'))\build\Common.prod.props" />
```

Comment:
so those Directory.Build.props are included automatically? Good to know

Reply:
Yup, see the section `Search scope` [here](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019)